src/sammich/plugins/create_issue.py

```diff
@@ -0,0 +1,47 @@
+import requests
+from dotenv import dotenv_values
+from machine.plugins.base import MachineBasePlugin
+from machine.plugins.decorators import command
+
+secrets = dotenv_values(".secrets")
+
+github_token = secrets["github_token"]
+
+
+class CreateIssuePlugin(MachineBasePlugin):
+    @command("/ghissue")
+    async def createissue(self, command):
+        channel_id = command._cmd_payload["channel_id"]
+        title = command.text.strip()
+
+        # Replace with your repository details
+        owner = "OWASP-BLT"
+        repo = "BLT-Sammich"
```

Comment:
These can be the class level attributes.

Or just inline them as you use them just once

It'd also make sense to make it configurable via settings if we want it to be easily used by others.